brainsets/core.py

```
@@ -133,3 +133,21 @@ def datetime_serialize_fn(obj, serialize_fn_map=None):
+
+_classes = [
+    "NestedEnumType",
+    "StringIntEnum",
+    "Dictable",
+]
+
+_constants = [
+    "serialize_fn_map",
+]
+
+__all__ = _functions + _classes + _constants
```

Ruff (0.15.7)

[error] Invalid format for `__all__`, must be `tuple` or `list`

(PLE0605)